src/utils/scripts.ts

```diff
@@ -4,6 +4,67 @@ import colors from "colors";
 import _ from "lodash";
 import { Logger } from "tslog";
 import { execaCommand } from "execa";
+import { debugLogger, log } from "./logging.js";
+
+export async function expandVariablesFromScript(
```

Comment:
Looks like this function expands only function URLs variables. I would name it to something more self explanatory. Maybe: `expandFunctionURLVariablesFromScript`

Reply:
Done